#磁导信息消息体
uint8 iNavType
int32 iRFID
uint8 iMLSEMotor
